admin/admin-service/src/main/java/io/mosip/admin/service/impl/AdminServiceImpl.java

```diff
@@ -73,5 +106,99 @@ private void createLostRidRequest(SearchInfo searchInfoRequest) {
 
 	}
 
+	public String getApplicantPhoto(byte[] isodata) throws Exception {
+		    ConvertRequestDto convertRequestDto = new ConvertRequestDto();
+		    convertRequestDto.setVersion("ISO19794_5_2011");
+			convertRequestDto.setInputBytes(isodata);
+			byte[] data = FaceDecoder.convertFaceISOToImageBytes(convertRequestDto);
+			String encodedBytes = new String(data);
+
+		return encodedBytes;
+	}
 
+	@Override
+	public LostRidDetailsDto getLostRidDetails(String rid) {
+		LostRidDetailsDto lostRidDetailsDto=new LostRidDetailsDto();
+		Map<String,String> lostRidDataMap=new HashMap<>();
+		FieldDtos fieldDtos=new FieldDtos();
+		RequestWrapper<FieldDtos> fieldDtosRequestWrapper=new RequestWrapper<>();
+		ConvertRequestDto convertRequestDto = new ConvertRequestDto();
+		try {
+			FieldResponseDto fieldResponseDto=new FieldResponseDto();
+			buildSearchFieldsRequestDto(fieldDtos,rid);
+			fieldDtosRequestWrapper.setRequest(fieldDtos);
+			ResponseWrapper<FieldDtos> fieldDtosResponseWrapper = restClient.postApi(ApiName.PACKET_MANAGER_SEARCHFIELDS, MediaType.APPLICATION_JSON,
```

Comment:
We are assigning the response to ResponseWrapper<FieldDtos>  but later we are trying to deserialize the response to FieldResponseDto. 
Why are we defining the FieldResponseDto again and again.